app.py

```diff
@@ -271,56 +271,49 @@ def convertPdfToText(event, context):
                 raise
 
     # Start converting PDF into JPEG Images
-    page_num = 0
+    n_pages = 0
     result = ''
     if os.path.isfile(pdf_file):
         app.log.info("Converting PDF to images!")        
-        pdf = pdfium.PdfDocument(pdf_file)
+        pdf = pdfium.PdfDocument(pdf_file, may_init_forms=True)
         version = pdf.get_version()  # get the PDF standard version
```

Comment:
as you don't actually need the pdf version, I'd rather remove that call? it's not overly relevant to know...